Assets/Tests/InputSystem/CoreTests_Editor.cs

```diff
@@ -743,6 +743,73 @@ public void Editor_InputAsset_CanReplaceBindingGroupThroughSerialization()
         Assert.That(map.bindings[0].groups, Is.EqualTo(""));
     }
 
+    struct AFTestConstants
```

Comment:
Is this "AF" capitalised prefix representing "Asset File" or what is it intended to represent? I would recommend renaming it and avoid "unknown" acronyms if this is the case and call it "AssetFileTestConstants"